### PR TITLE
Add onKeyDown and onKeyPress event handlers to the input field

### PR DESCRIPTION
**Pull request by antoniopuero**
### Checklist

- [x] All tests passing
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary
- [x] Commits and PR follow conventions

### CODE REVIEW

File: src/input.jsx [hunk 1]
```diff
@@ -42,9 +42,17 @@ class Input extends React.Component {
   }
 
   /**
-   * When a key gets pressed in the input
+   * When the input loses focus
```
Comment (yfr) [inline src/input.jsx]: Is this really the Description for an keyPress event?

**Comment by yfr** [comment]
Thank you for your PR! But I'm not completely fine with the fact that we have the `keyDown` event that is handled in two areas of the component. @Scarysize what do you think?

Anyway, the new props need some documentation. And it should be clearly mentioned that we have this special case for event propagation in the component. I would suggest a whole new paragraph in the readme.

**Comment by antoniopuero** [comment]
heh, man, why did you ask for pull request then if you aren't sure about it?

**Comment by yfr** [comment]
First, I did not ask for a PR, my colleague did. We work at the same company but are different persons with different opinions. I just put my concerns in here.
Second, from my understanding @ro-ka said that the `keyDown` event is already in use but the keyPress event could be used. And that's the same thought that I addressed.

**Comment by antoniopuero** [comment]
I've deleted onKeyDown event handler, onKeyPress has left

**Comment by yfr** [comment]
Hey @antoniopuero. Thank you for the changes. That looks very good to me. Can you add a description to the `README.md` please?

**Comment by antoniopuero** [comment]
Readme has been updated, translation could be weak though

**Comment by yfr** [comment]
Ok for me. @Scarysize it's your turn. Merge when ready. 

@antoniopuero Thanks again!

**Comment by antoniopuero** [comment]
no worries, sorry for being a douchebag :)

**Comment by Scarysize** [comment]
Thanks!

**Pull request merged**